.github/ISSUE_TEMPLATE/improve-docs.yml

```
@@ -0,0 +1,42 @@
+name: Improve content
+description: Make a suggestion to improve the content in an existing article.
```

Small thing but do we call them `articles` or `pages`? I've normally seen `page` for docs site.

---

I can get behind using "page!"
```suggestion
description: Make a suggestion to improve the content in an existing page.
```